# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/workers/WorkerScope.h
# Commit: d801fdeac90b
# Full Hash: d801fdeac90bc63de7f7efd090ced48bf871121a
# Author: Tom Ritter <tom@mozilla.com>
# Date: 2022-11-29 08:40:32
# Regressor Bug: 1778510
# File Overlap Count: 1
# Description:
#   Bug 1778510: Add RTPCallerType to GlobalObject r=asuth
#   
#   This centralizes the logic in one place.
#   
#   In order to do this, we will need to check the principal
# ==============================================================================

diff -r 5af032c628c0 -r d801fdeac90b dom/workers/WorkerScope.h
--- a/dom/workers/WorkerScope.h	Mon Nov 28 18:04:06 2022 +0000
+++ b/dom/workers/WorkerScope.h	Mon Nov 28 18:04:07 2022 +0000
@@ -173,6 +173,8 @@
  protected:
   ~WorkerGlobalScopeBase();
 
+  bool IsSystemPrincipal() const final;
+
   CheckedUnsafePtr<WorkerPrivate> mWorkerPrivate;
 
   void AssertIsOnWorkerThread() const {